protocol/rpcprovider/rpcprovider.go

```diff
@@ -361,6 +371,27 @@ func (rpcp *RPCProvider) SetupEndpoint(ctx context.Context, rpcProviderEndpoint
 		return utils.LavaFormatError("[PANIC] failed to RegisterForSpecUpdates, panic severity critical error, aborting support for chain api due to invalid chain parser, continuing with others", err, utils.Attribute{Key: "endpoint", Value: rpcProviderEndpoint.String()})
 	}
 
+	// warn if not all internal paths are configured
+	configuredInternalPaths := GetAllNodeUrlsInternalPaths(rpcProviderEndpoint.NodeUrls)
+	chainInternalPaths := chainParser.GetAllInternalPaths()
+	nonConfiguredInternalPaths := lavaslices.Difference(chainInternalPaths, configuredInternalPaths)
```

Comment:
if the provider defines the base path, we should be able to add all internal paths (because they are all relative to the base)
we need to maintain the auto generation of urls we had so far